### ISSUE TITLE
How to fill up the whole map with with hex grid?

### ISSUE THREAD

**Issue by tadpolema** [Turfjs/turf#2199]
I try to use function 'hexgrid' to fill the whole world, with bbox = [-180, -90, 180, 90], mask is the featureCollection with coordinates from -180 to 180. But what I got from 'hexgrid' is an empty Array.-> {type: "FeatureCollection", features: Array(0)}. I wonder how to get the proper fill-map. Thanks!

**Comment by JamesLMilner** [comment]
I tried to do this myself but struggled to find a satisfactory result. Maybe @rowanwins would know best?

**Comment by punkish** [comment]
I can attest that `hexGrid` seems to behave very strangely. When given a `bbox = [-180, -90, 180, 90]`, I get no features at all. Interestingly, if I use turf v 2.0 directly from mapbox [api.tiles.mapbox.com/mapbox.js/plugins/turf/v2.0.0/turf.min.js](//api.tiles.mapbox.com/mapbox.js/plugins/turf/v2.0.0/turf.min.js), I can create a hex grid for the entire world.

```js
const bbox = [-179.84, -60, 179, 74];
const cellSide = 1;
const units = 'kilometers';
const hexgrid = turf.hexGrid(bbox, cellSide, 'kilometers');

// using Leafletjs
L.geoJSON(hexgrid).addTo(map); // this works
```

Strangely, I have set the `cellSide = 1` which makes no sense with `kilometers` units. Of course, the cells are not 1 km in size (there would be millions of them for the whole world, and my browser would crash). But making the cell size bigger makes them too big. At cell size of 1000, one cell covers the entire world. I discovered this when searching for answers I came across https://github.com/muxlab/map-effects-100/blob/gh-pages/Leaflet/10_hex-grid-with-turf.html

Whatever the change from v 2.0.0 to the current version 6.x, seems like `hexGrid` has gone through some changes that have not been documented clearly.

Would be nice if someone can shed more light on what is going on
